TicketsDemo.Domain/DefaultImplementations/LoggingDecor.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Configuration;
+using TicketsDemo.Data.Entities;
+using TicketsDemo.Domain.Interfaces;
+
+namespace TicketsDemo.Domain.DefaultImplementations
+{
+    public class LoggingDecor : IReservationService
+    {
+        private IReservationService _decorObject;
+        private ILogger _logger;
+        private string _path;
+
+        public LoggingDecor(IReservationService reservation, ILogger logger)
+        {
+            _decorObject = reservation;
+            _logger = logger;
+            _path = $"{AppDomain.CurrentDomain.BaseDirectory}{ConfigurationManager.AppSettings["LogFile"]}"; 
```

Comment:
_path is not used

Reply:
Inside the logger, it would be better to read path from separate ISettings service or smth like that. Where ISettings will return all configurable settings of your app, such as logging file path.

interface ISettings {
     string LogFilePath {get;set;}
}